tests/wpt/mozilla/tests/css/inline_block_percentage_margin.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <link rel="match" href="inline_block_percentage_margin_ref.html">
+        <style>
+         body {
+             width: 500px;
+         }
+         span {
+             display: inline-block;
+             margin: 0 10%;
+         }
+        </style>
+    </head>
+    <body>
+        <span>Foo</span>
```

Comment:
I think if not needed, it's better to use solid boxes instead of tests for reftests, but your call.